tools/earthly.sh

```diff
@@ -36,7 +36,7 @@ if ! is-file "$EARTHLY_EXE"; then
 fi
 
 run-earthly() {
-    "$EARTHLY_EXE" "$@"
+    "$EARTHLY_EXE" --buildkit-image 901841024863.dkr.ecr.us-east-1.amazonaws.com/dockerhub/earthly/buildkitd:v0.8.3 "$@"
```

Comment:
In the suggested refactor, specifying `--buildkit-image=...` would need to be done in the generated `tools/earthly.sh` commands for the EVG tasks, rather than hardcoded in this file.